src/pages/dashboards/Users/FilterBar.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  ButtonGroup,
+  ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem
+} from "reactstrap";
+
+const FilterBar = ({changeItemsFilter, changeItemsMeta, currentFilter, itemsMeta}) => {
+
+	const [maxItemsOpen, setMaxItems] = useState(false);
+	const maxToggle = () => {setMaxItems(!maxItemsOpen); console.log(maxItemsOpen)}
+
+	const handleFilterSelect = (value) => {
+	  const { name, val } = value;
+	  if (currentFilter[name] !== val) {
+	    let newFilters = {...currentFilter, [name]: val}
+	    changeItemsFilter(newFilters);
+	  }
+	}
+
+	const handleItemsMetaChange = meta => {
+		const { name, val } = meta;
+		if (itemsMeta[name] !== val) {
+		  let newMeta = {...itemsMeta, [name]: val}
```

Comment:
You don't need to pass the name inside this method because it's the same for each call.
```javascript
let newMeta = {...itemsMets, maxItems: value};